program/caffe-time-opencl/caffe.cpp

```diff
@@ -268,12 +286,12 @@ int train() {
   }
 
   if (gpus.size() > 1) {
-    caffe::P2PSync<float> sync(solver, NULL, solver->param());
-    std::vector<device*> devices;
-    for (int_tp i = 0; i < gpus.size(); ++i) {
-      devices.push_back(Caffe::Get().GetDevice(i, true));
-    }
-    sync.Run(devices);
+//    caffe::P2PSync<float> sync(solver, NULL, solver->param());
```

Comment:
I'll uncomment and fix it according to appropriate API in scope of 
issues #37 a little bit later